google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -32,12 +32,20 @@ interfaces:
   retry_params_def:
   - name: default
     initial_retry_delay_millis: 100
-    retry_delay_multiplier: 1.2
-    max_retry_delay_millis: 1000
-    initial_rpc_timeout_millis: 2000
-    rpc_timeout_multiplier: 1.5
-    max_rpc_timeout_millis: 30000
-    total_timeout_millis: 45000
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000 # 60 seconds
+    initial_rpc_timeout_millis: 12000 # 12 seconds
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 12000 # 12 seconds
+    total_timeout_millis: 600000 # 10 minutes
+  - name: messaging
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000 # 60 seconds
```

Comment:
In the doc comments, Emilio and Haakon suggest a lower timeout for Publish/Pull/Ack (10s)